src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
```

Comment:
Consider adding a more descriptive class name for better readability and consistency. 📚